editor/plugins/item_list_editor_plugin.cpp

```diff
@@ -127,6 +127,13 @@ void ItemListPlugin::_get_property_list(List<PropertyInfo> *p_list) const {
 	}
 }
 
+void ItemListPlugin::_bind_methods() {
+	// These are used to bind the function handles used in the undo_redo methods to object methods for the plugins
+	ClassDB::bind_method(D_METHOD("undoRedo_add_new_item", "idx"), &ItemListPlugin::undoRedo_add_new_item);
+	ClassDB::bind_method(D_METHOD("undoRedo_re_add_item", "name", "position", "icon", "is_enabled"), &ItemListPlugin::undoRedo_re_add_item);
+	ClassDB::bind_method(D_METHOD("undoRedo_erase", "p_idx"), &ItemListPlugin::undoRedo_erase); 
```

Comment:
`undoRedo_*` method name is a bit weird to me. Undo/redo related methods are not prefixed with `undoRedo` anywhere else in the codebase as far as I can see. Additionally, this is a mix of camelCase and snake_case, which should not be mixed. Classes are PascalCase and methods and members are generally snake_case.